pom.xml

```diff
@@ -103,7 +116,7 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-shade-plugin</artifactId>
-          <version>3.2.3</version>
+          <!-- Version defined in Apache parent pom -->
```

Comment:
let's keep it there to not be unstable due to "external" parent pom

Reply:
no problem, I'll go back through and specify the versions

Reply:
fixed in new commit. plugin versions are defined under _pluginManagement_ in main pom